modules/dnn/src/layers/nary_eltwise_layers.cpp

```diff
@@ -954,6 +904,28 @@ class NaryEltwiseLayerImpl CV_FINAL : public NaryEltwiseLayer
         return Ptr<BackendNode>(new InfEngineNgraphNode(node));
     }
 #endif
+
+#ifdef HAVE_VULKAN
+    virtual Ptr<BackendNode> initVkCom(const std::vector<Ptr<BackendWrapper> > &inputs,
+                                       std::vector<Ptr<BackendWrapper> > &outputs) CV_OVERRIDE
+    {
+        std::vector<Ptr<VkComBackendWrapper>> inputWrappers;
+        std::vector<MatShape> inputShapes;
+        std::transform(inputs.begin(), inputs.end(), std::back_inserter(inputWrappers), [] (const Ptr<BackendWrapper>& w) { return w.dynamicCast<VkComBackendWrapper>(); });
+        for (const auto &ptr: inputWrappers) {
+            CV_Assert(ptr);
+        }
+        std::transform(inputWrappers.begin(), inputWrappers.end(), std::back_inserter(inputShapes), [] (const Ptr<VkComBackendWrapper>& w) { return shape(*(w->getMat())); });
+        auto outputWrapper = outputs[0].dynamicCast<VkComBackendWrapper>();
+        CV_Assert(outputWrapper);
+        auto outputShape = shape(*(outputWrapper->getMat()));
+        std::vector<Mat> vkBlobs;
```

Comment:
Are these variables `InputWrappers`, `inputShapes`, `outputWrapper`, `outputShape` and `vkBlobs` used anywhere?

Reply:
Seems useless. They are remaining codes from a previous version in which `NaryEltwiseHelper` is not introduced.